front/src/Api/Events/JoinMucEvent.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+
+export const isJoinMucEvent = z.object({
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?